### PR TITLE
Added NJ committee scraper

### PR DESCRIPTION
**Pull request by divij-sinha**
New committee scraper for New Jersey written in spatula.

Makes calls to four different APIs for Joint, Senate, Assembly, and Other Committees, respectively. Parses the JSON responses using `JsonPage` class to yield committee objects.

### CODE REVIEW

**Comment by chrisyamas** [comment]
*Note: New Committee Scraper written by attendee of Open States Summit*

File: scrapers_next/nj/committees.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                members = com_membership['Committees'][com['Comm_Status']]
+            except KeyError:
+                members = []
+            for member in members:
+                name = member['FullName']
+                name_regex = re.compile("(.*?), +(.*)")
+                name_match = name_regex.match(name)
+                new_name = " ".join([name_match.group(2), name_match.group(1)])
+                role = member['Position']
+                if role != "":
```
Comment (johnseekins) [inline scrapers_next/nj/committees.py:32]: Generally, python encourages using `if not role:` for empty strings. Unless we are okay with `None` being allowed past this gate (i.e. `"" != None`).

**Review by chrisyamas — APPROVED** [review]
Hi @divij-sinha, this is a great contribution, and especially so as a first-time contributor to our repo! We're thrilled to have you as a part of the Open States community.

Apologies for taking so long to get to reviewing your PR, and I also hope you'll forgive me for just making the last bit of needed modifications myself before approving and merging. We typically would communicate/request changes and give you time to push those commits to the PR in review, but we're on a time crunch with getting this session's committee data imported and there are several nuances that needed.

**Comment by divij-sinha** [comment]
Thanks @chrisyamas! Apologies for not seeing this earlier and making some of the flagged changes. Glad to be able to contribute, and hoping that I can do this more often!

**Pull request merged**